io_scene_dae/export_dae.py

```diff
@@ -556,7 +542,7 @@ def export_mesh(self, node, armature=None, skeyindex=-1, skel_source=None,
         if armature is not None:
             si = self.skeleton_info[armature]
 
-        has_tangents = self.config["use_tangent_arrays"]  # could detect..
+        has_tangents = self.config["use_tangent_arrays"]  # TODO: Review
```

Comment:
I reviewed the line myself. In short, it looks up a user-set bool property, "Export Tangent and Binormal arrays (for normalmapping)". It can be either True or False, not much to wonder there except that perhaps the original author was wondering if the property could be detected in another way rather than letting the user set it. I think it could be for the best if the "TODO: Review" comment was removed altogether - the line itself is fine.